nodeJS/final_project/project_odin_book.md

```diff
@@ -43,7 +43,7 @@ The following requirements are a very global list of features your app should ha
 
 #### Extra credit
 
-1. Make posts also allow images (either just via a URL or by uploading one.)
+1. Make posts also allow images (either just via a URL or by uploading one). If you did the extra credit from [Inventory Application Project](https://www.theodinproject.com/lessons/nodejs-inventory-application) then you may recall [Cloudinary] being a good option for hosting user-uploaded images. The URLs Cloudinary provides you can then be stored in your database instead of the raw image binary data.
```

Comment:
Is the first `[Cloudinary]` supposed to be a link?

Reply:
Oops.